RusimbiPatrick/Prolite
# Created User Interface for Property Pro-Lite

What does this PR do?
**What does this PR do?**

Create a full UI for Propertety Pro-Lite
**Description of Task to be done?**
i. User can sign up. 
 ii. User can sign in. 
 iii. User (agent) can post a property advert. 
 iv. User (agent) can update the details of a property advert.  
v. User (agent) can mark his/her posted advert as sold.  
vi. User (agent) can delete an advert.  
vii. User can view all properties.  
viii. User can view all properties of a specific type - 2 bedroom, 3 bedroom,  mini flat etc.  ix. User can ix. ix.User can view a specific property

**How should this be manually tested?**
Visit these pages to check if the tasks mentioned above function in the current UI

Description of Task to be done?

Link all admin and cashier pages with the user's dashboard
for easy Demonstration
How should this be manually tested?
Visit these pages to check if the tasks mentioned above function in the current UI
